docs/UserGuide.md

```diff
@@ -3,20 +3,33 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
-
-* Table of Contents
-{:toc}
+NUSpam is a desktop app for managing contacts **targeted at marketers who require fast manipulation and precise handling of contact data.** It enables marketers to more easily manage and make use of email and phone leads, and **minimise tedious and repetitive tasks** such as data entry, email blasts, and mail merge.
+
+* [Quick start](#quick-start)
+* [Features](#features)
+  * [Viewing help: `help`](#viewing-help-help)
+  * [Adding a person: `add`](#adding-a-person-add)
+  * [Batch importing contacts: `import`](#batch-importing-contacts-import)
+  * [Listing all persons: `list`](#listing-all-persons-list)
+  * [Editing a person: `edit`](#editing-a-person-edit)
+  * [Locating persons by name: `find`](#locating-persons-by-name-find)
+  * [Deleting a person: `delete`](#deleting-a-person-delete)
+  * [Clearing all entries: `clear`](#clearing-all-entries-clear)
+  * [Exiting the program: `exit`](#exiting-the-program-exit)
+  * [Saving the data](#saving-the-data)
+  * [Editing the data file](#editing-the-data-file)
+* [FAQ](#faq)
+* [Command Summary](#command-summary)
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Quick start
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `NUSpam.jar` from [here](https://github.com/AY2122S1-CS2103T-W13-2/tp/releases).
```

Comment:
`NUSpam.jar` doesn't exist yet, maybe can leave a comment here to say something like "coming soon" or just lead to `v0.1` `addressbook.jar`. We can change to this when we actually release something.

Reply:
I removed the link, and labeled it as "(Coming Soon)"